src/app/services/templates.service.ts

```diff
@@ -127,15 +135,27 @@ export class TemplatesService {
 }
 
 function isInstanceOfPluginFilterOr(pluginFilter: PluginFilterExpr): pluginFilter is PluginFilterOr {
-    return pluginFilter != null && (pluginFilter as PluginFilterOr).or !== undefined;
+    if (pluginFilter != null) {
+        const _or = (pluginFilter as PluginFilterOr).or;
```

Comment:
To be extra safe we could also ensure that no other keys are present on the object using `Object.keys(pluginFilter)`. Otherwise there could be a filter that is both an and and an or.